src/legacy/core_plugins/kibana/server/tutorials/apm/instructions/apm_agent_instructions.js

```diff
@@ -304,52 +304,66 @@ export const createRackAgentInstructions = (apmServerUrl = '', secretToken = '')
+    textPre: i18n.translate('kbn.server.tutorials.apm.jsClient.installDependency.textPre', {
+      defaultMessage: 'You can install the Agent as a dependency to your application with \
+`npm install @elastic/apm-rum --save`.\n\n\
+The Agent can then be initialized and configured in your application like this:',
     }),
     commands: `import {curlyOpen} init as initApm {curlyClose} from '@elastic/apm-rum'
 var apm = initApm({curlyOpen}
 
-  // ${i18n.translate('kbn.server.tutorials.apm.jsClient.configure.commands.setRequiredServiceNameComment', {
+  // ${i18n.translate('kbn.server.tutorials.apm.jsClient.installDependency.commands.setRequiredServiceNameComment', {
     defaultMessage: 'Set required service name (allowed characters: a-z, A-Z, 0-9, -, _, and space)',
   })}
   serviceName: '',
 
-  // ${i18n.translate('kbn.server.tutorials.apm.jsClient.configure.commands.setCustomApmServerUrlComment', {
+  // ${i18n.translate('kbn.server.tutorials.apm.jsClient.installDependency.commands.setCustomApmServerUrlComment', {
     defaultMessage: 'Set custom APM Server URL (default: {defaultApmServerUrl})',
     values: { defaultApmServerUrl: 'http://localhost:8200' },
   })}
   serverUrl: '${apmServerUrl}',
 
-  // ${i18n.translate('kbn.server.tutorials.apm.jsClient.configure.commands.setServiceVersionComment', {
-    defaultMessage: 'Set service version (required for sourcemap feature)',
+  // ${i18n.translate('kbn.server.tutorials.apm.jsClient.installDependency.commands.setServiceVersionComment', {
+    defaultMessage: 'Set service version (required for source map feature)',
   })}
   serviceVersion: ''
 {curlyClose})`.split('\n'),
-    textPost: i18n.translate('kbn.server.tutorials.apm.jsClient.configure.textPost', {
-      defaultMessage: 'See the [documentation]({documentationLink}) for advanced usage.',
-      values: { documentationLink: '{config.docs.base_url}guide/en/apm/agent/rum-js/current/index.html' },
+  },
+  {
+    title: i18n.translate('kbn.server.tutorials.apm.jsClient.scriptTags.title', {
+      defaultMessage: 'Set up the Agent with Script Tags',
+    }),
+    textPre: i18n.translate('kbn.server.tutorials.apm.jsClient.scriptTags.textPre', {
+      defaultMessage: 'Alternatively, you can use Script tags to set up and configure the Agent. \
+Add a `<script>` tag to the HTML page and use the `elasticApm` global object to load and initialize the agent. \
+Don\'t forget to download the latest version of the RUM Agent from [GitHub]({GitHubLink}) or [UNPKG]({UnpkgLink}), \
+and host the file on your Server/CDN before deploying to production.',
+      values: {
+        GitHubLink: 'https://github.com/elastic/apm-agent-rum-js/releases/latest',
+        UnpkgLink: 'https://unpkg.com/@elastic/apm-rum/dist/bundles/elastic-apm-rum.umd.min.js'
+      },
     }),
+    commands: `\
+<script src="https://your-cdn-host.com/path/to/elastic-apm-rum.umd.min.js" crossorigin></script>
+<script>
+  elasticApm.init({curlyOpen}
+    serviceName: '',
```

Comment:
serviceName is a mandatory field, can you change it to `app-name` or something similar.